aten/src/ATen/native/cuda/ScatterGatherKernel.cu

```diff
@@ -536,9 +536,6 @@ void scatter_fill_cuda_kernel(const Tensor& self, int64_t dim, const Tensor& ind
 }
 
 void scatter_add_cuda_kernel(const Tensor& self, int64_t dim, const Tensor& index, const Tensor& src) {
-  // See Note [Writing Nondeterministic Operations]
-  // Nondeterministic because of atomicAdd usage
-  globalContext().alertNotDeterministic("scatter_add_cuda_kernel");
```

Comment:
assuming it's unlikely someone trips over this in the future by sending floating point tensors to it while expecting determinism